src/core_ocean/shared/mpas_ocn_tracer_short_wave_absorption_variable.F

```diff
@@ -196,6 +200,23 @@ subroutine ocn_tracer_short_wave_absorption_variable_tend(meshPool, swForcingPoo
       !$omp end do
       !$omp end parallel
 
+      if(config_enable_shortwave_energy_fixer) then!
+#ifndef CPRPGI!
+         !$omp parallel!
+         !$omp do schedule(runtime) private(k, layerFraction)!
+#endif!
+         do iCell = 1, nCells!
+            layerFraction = bottomlayerShortwaveTemperatureFlux(iCell) / (maxLevelCell(iCell) + 1.0E-15_RKIND)!
+            do k=1,maxLevelCell(iCell)!
+               tend(index_temperature, k, iCell) = tend(index_temperature, k, iCell) + layerFraction!
+            end do!
+         end do!
+#ifndef CPRPGI!
+         !$omp end do!
+         !$omp end parallel!
+#endif
```

Comment:
@mattdturner could you check this loop as well?

Reply:
This looks good as well